aiogram/types/reply_keyboard.py

```diff
@@ -90,6 +90,16 @@ def insert(self, button):
             self.add(button)
         return self
 
+    def __iadd__(self, other):
+        return self.add(other)
+
+    def __add__(self, other):
+        return ReplyKeyboardMarkup(
```

Comment:
`self.__class__(...)` - it would be clearer

Reply:
I don't think so. Explicit class initializing is better then implicit

Reply:
It's a self class. In this case you know it's behavior without knowing any class names. You shouldn't remember: is it the same class or not.
So it's simple and explicit.

Reply:
done